dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -707,13 +731,17 @@ void AnimatedVisualPlayer::SetProgress(double progress)
         return;
     }
 
+    // Make sure that animations are instantiated.
+    InstantiateAnimations();
+
     auto clampedProgress = std::clamp(static_cast<float>(progress), 0.0F, 1.0F);
 
     // WARNING: Reentrance via IsPlaying DP may occur from this point down to the end of the method
     //          iff m_nowPlaying.
 
     // Setting the Progress value will stop the current play.
     m_progressPropertySet.InsertScalar(L"Progress", static_cast<float>(clampedProgress));
+    m_lastPlayProgress = progress;
```

Comment:
>m_lastPlayProgress = progress [](http://example.com/codeflow?start=4&length=29)

m_lastPlayProgress will be clamped later when it is used? (i.e. in stop?)

Reply:
ah, probably it is better to set it to `clampedProgress` here, thx